models.py

```diff
@@ -272,6 +274,8 @@ class PayDbTransaction(db.Model):
     def __init__(self, action, sender, recipient, amount, attachment):
         self.token = secrets.token_urlsafe(8)
         self.timestamp = int(time.time())
+        #self.date = datetime.datetime.now()
+        self.date = datetime.datetime.fromtimestamp(self.timestamp)
```

Comment:
it we want to change from a timestamp to a datetime type we should just remove the timestamp field